.github/scripts/get-changelog-from-change-txt.js

```diff
@@ -0,0 +1,35 @@
+'use strict';
+
+const marked = require("marked");
+const fs = require('fs');
+const { VERSION } = process.env;
+
+if (!VERSION) {
+	console.error('missing VERSION env var');
+	process.exit(1);
+	return;
+}
+
+for (let fileName of ['changelog', 'readme'] ) {
+	(async () => {
```

Comment:
Why wrap in a function (and why `async`)?

Reply:
This is copied from the pro repo as well.
I assume it's wrapped in a function in order for the `async` to be available.
Not sure why the `async` was needed in the first place though....

Reply:
Can we fix & align both repos?

Reply:
Done
Opening a ticket and a PR for the Pro repo